src/Avalonia.Controls/ToolTip.cs

```diff
@@ -238,6 +242,9 @@ private void Open(Control control)
             
             _popup.ConfigurePosition(control, GetPlacement(control), 
                 new Point(GetHorizontalOffset(control), GetVerticalOffset(control)));
+
+            WindowManagerAddShadowHintChanged(_popup, false);
```

Comment:
@danwalmsley 
Because ToolTip is not based on Popup (as it do in UWP or AutoCompleteBox in Avalonia), I can't just set WindowManagerAddShadowHint property on it, so I duplicated part of that code.